src/test/java/hudson/tasks/junit/JUnitResultArchiverTest.java

```diff
@@ -100,13 +100,35 @@
 		wc.getPage(build, "testReport/hudson.security"); // package
 		wc.getPage(build, "testReport/hudson.security/HudsonPrivateSecurityRealmTest/"); // class
 		wc.getPage(build, "testReport/hudson.security/HudsonPrivateSecurityRealmTest/testDataCompatibilityWith1_282/"); // method
-		
-
 	}
 
+    @LocalData
+    @Test public void basicSuiteArchiveId() throws Exception {
+        archiver.setArchiveId("dasId");
+        FreeStyleBuild build = project.scheduleBuild2(0).get(10, TimeUnit.SECONDS);
+
+        assertTestResults(build);
+
+        WebClient wc = j.new WebClient();
+        wc.getPage(project); // project page
+        wc.getPage(build); // build page
+        wc.getPage(build, "testReport");  // test report
+        wc.getPage(build, "testReport/hudson.security"); // package
+        wc.getPage(build, "testReport/hudson.security/HudsonPrivateSecurityRealmTest/");// class
+        HtmlPage methodPage = wc.getPage(build, "testReport/hudson.security/HudsonPrivateSecurityRealmTest/testDataCompatibilityWith1_282/");// method
+        //OK good it looks like before
+
+        //Now what about the suite archive id?
+        j.assertStringContains(methodPage.asText(), "[dasId]");
+        TestResultAction testResultAction = build.getAction(TestResultAction.class);
+        SuiteResult suite = testResultAction.getResult().getSuite("hudson.security.HudsonPrivateSecurityRealmTest");
+        assertNotNull(suite);
+        assertEquals("dasId", suite.getArchiveId());
+    }
+
     @RandomlyFails("TimeoutException from basic")
-   @LocalData
-   @Test public void slave() throws Exception {
+    @LocalData
+    @Test public void slave() throws Exception {
```

Comment:
Inherited sin. put the `@Test` annotation on a new line before the `public` modifier